pkg/upgrade/control_plane.go

```diff
@@ -700,3 +726,23 @@ func updateKubeadmKubernetesVersion(original *v1.ConfigMap, version string) (*v1
 
 	return cm, nil
 }
+
+func removeNodeFromClusterStatus(original *v1.ConfigMap, nodeName string) (*v1.ConfigMap, error) {
+	cm := original.DeepCopy()
+
+	clusterStatus := &kubeadmv1beta2.ClusterStatus{}
```

Comment:
we're using v1beta1 types from CABPK, is there any reason you imported v1beta2 instead?